src/traces/sankey/render.js

```diff
@@ -701,13 +744,17 @@ module.exports = function(gd, svg, calcData, layout, callbacks) {
         })
         .style('stroke-width', function(d) {
             return salientEnough(d) ? d.linkLineWidth : 1;
-        });
+        })
+        .attr('d', linkPath());
 
-    sankeyLink.transition()
-       .ease(c.ease).duration(c.duration)
-       .attr('d', linkPath());
```

Comment:
Animating the SVG path for links would look odd. Transitioning from a normal link to a circular link one for example would look broken. It makes more sense for the links to disappear and reappear, hence the change here...

Reply:
Nicely done!